packages/core/src/components/hero/hero.css

```diff
@@ -0,0 +1,252 @@
+}
+.hds-hero__image-below-koros {
+  display: block;
+}
+.hds-hero__image-below-koros .hds-hero__image {
+  object-fit: cover;
+  height: 100%;
+  object-position: var(--image-position);
+}
+.hds-hero--with-background__container {
+  position: relative;
+}
+.hds-hero--with-background__background {
+  width: 100%;
+}
+.hds-hero--with-background__background .hds-hero__image {
+  object-fit: cover;
+  height: 100%;
+  object-position: var(--image-position);
+}
+
+.hds-hero--with-background__contentColumns {
+  display: flex;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.hds-hero--with-background__empty-column {
+  display: none;
+}
+
+.hds-hero--background-image .hds-hero__content {
+  padding: 0;
+}
+
+.hds-hero--background-image .hds-hero__card {
+  background-color: var(--background-color);
+  padding: var(--padding-vertical) var(--padding-horizontal);
+}
+.hds-hero--background-image__top-koros {
+  display: block;
+}
+.hds-hero--background-image-bottom-koros {
+  display: none;
+}
+
+.hds-hero--diagonal-koros .hds-hero--with-background__container {
+  display: flex;
+  position: relative;
+  overflow: hidden;
+  z-index: 1;
+  flex-direction: column;
+}
+.hds-hero--diagonal-koros .hds-hero__content {
+  display: block;
+  z-index: 2;
+  width: 100%;
+  padding: 0;
+  box-sizing: border-box;
+}
+.hds-hero--diagonal-koros .hds-hero__card {
+  background-color: var(--background-color);
+  padding: var(--padding-vertical) var(--padding-horizontal);
+}
+.hds-hero--diagonal-koros .hds-hero--diagonal-koros__koros-and-background {
+  transform: scaleY(-1);
+  overflow: hidden;
+}
+
+.hds-hero--with-background__background,
+.hds-hero__image-below-koros {
+  height: 300px;
+}
+@media only screen and (min-width: 768px) {
+  .hds-hero {
+    --padding-horizontal: var(--horizontal-padding-medium);
+    --padding-vertical: var(--spacing-layout-m);
+  }
+  .hds-hero--with-background__background,
+  .hds-hero__image-below-koros {
+    height: 400px;
+  }
+}
+@media only screen and (min-width: 992px) {
+  .hds-hero {
+    --padding-horizontal: var(--horizontal-padding-large);
+    --padding-vertical: var(--spacing-layout-xl);
+  }
+  .hds-hero p {
+    font-size: var(--fontsize-body-xl);
+  }
+  .hds-hero--image-left .hds-hero__image-below-koros,
+  .hds-hero--image-right .hds-hero__image-below-koros {
+    display: none;
+  }
+  .hds-hero__content--two-columns {
+    flex-direction: row;
+  }
+  .hds-hero__content--two-columns > * {
+    width: 50%;
+  }
+  .hds-hero__content--two-columns > *:first-child {
+    margin-right: var(--spacing-xl);
+  }
+  .hds-hero__content--two-columns > *:last-child {
+    margin-left: var(--spacing-xl);
+  }
+  .hds-hero__content--two-columns__image-container {
+    display: block;
+  }
+  .hds-hero--background-image .hds-hero--with-background__container {
+    padding: var(--padding-vertical) var(--padding-horizontal);
+  }
+
+  .hds-hero--background-image .hds-hero__content {
+    --padding-vertical: var(--spacing-2-xl);
```

Comment:
Yes, the box in the background image -variant is aligned differently with padding.

> I was just thinking should the overrides happen on the same hds-hero level as is done in media queries

I'm sure what that line means :)

Reply:
I meant that should it be enough to use
```css
.hds-hero--background-image  {
```
 😁

Reply:
No, it should affect only the `.hds-hero__content`. If it set to `.hds-hero--background-image` it will affect also `.hds-hero--background-image .hds-hero--with-background__container`